### ISSUE TITLE
Tensorflow

### ISSUE THREAD

**Issue by anantgupta129** [tensorflow/community#290]
# Community repo

I am working on cnn model which has 4 conv layers and 3 dense layers. dataset have around 28000 images and 7000 test images. The model has saved checkpoints and I have trained it several times and achieved 60 % accuracy so far, and while training learning rate is reduced to 2.6214403e-07 (as i used ReduceLROnPlateau factor 0.4). I have question if I increased the learning rate say 1e-4. and resumed the training how will it effect my model? Is It a good idea?

**Comment by therc01** [comment]
Hi, please assign me this issue.

**Comment by mihaimaruseac** [comment]
This should be asked on StackOverflow, or elsewhere. The community repo is about comunity organisation and RFCs.
